### PR TITLE
fix accessibility issues

### PR DESCRIPTION
**Pull request by HanaSabih**
Performed following accessibility checks on portfolio project

1. Page title -meaningful title  already exists
2. Image text alternatives - all images in the project have text alternatives.
3. Text headings - All sections in the project have titles
4. Color contrast - Color Contrast was determined by the Figma design
5. Resize - All text contents in the project are zoomable and have no distortions
6. Interaction - Input form elements are labeled and hidden in the mobile version and displayed in desktop versions per the FIGMA design template.
7. Moving content - the webpage has no moving 
8. Multimedia - the web page has no multimedia(audio/video)
9. Basic structure of the page and the navigation menu links are meaningful and self-explanatory

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,14 +18,14 @@ <h5>Welcome</h5>
...
           </div>
         </nav>
       </header>
-      <section class="headline">
-        <h1>
+      <section title="headline" class="headline">
+        <h1 id="headline">
           Hey there. I'm Hana<br />
           <span>I'm a software developer</span>
         </h1>
```
Comment (iambenkis) [inline index.html:29]: - [ ] Good job with this `h1` tag here :+1: , Keep doing the good work :+1:

File: index.html [hunk 2]
```diff
@@ -214,6 +216,7 @@ <h2>
           maxlength="30"
           required
         />
+        <label for="email" class="disnone">Email address </label>
         <input
```
Comment (iambenkis) [inline index.html:218]: - [ ] Good job with this `input`  labelled here :+1: , keep doing the good work :+1:

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @HanaSabih 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized upto 200%



 ## Required Changes ♻️

- [ ] Good job integrating the accessibility in your website, however make sure you have the ` alternativea`  here, it will make your website more accessible since your portfolio will be viewed by different people.
Screenshot from 2022-12-13 18-37-08

- [ ] Kindly remove this `h5` inside the `a` tag , it will make your website respect the heading
Screenshot from 2022-12-13 18-50-38

- [ ] Make sure you have here an `h2` tag because you already have an `h1` before it :+1: 
Screenshot from 2022-12-13 18-53-04


_Check the button under the review_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @HanaSabih and team,

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested change has been implemented

### Note:
- This is a pair programming project as stated by the [submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md#work-and-submission-mode), however, only one of the team members has made commits. Kindly ensure that in subsequent pair programming activities, all the members of the team contribute to the project evidenced by their commits. I have waved this requirement because the previous reviewer seemed to have missed that out.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**